openpype/settings/defaults/project_settings/global.json

```diff
@@ -243,6 +243,15 @@
                     "tasks": [],
                     "template": "{family}{variant}"
                 },
+                {
+                    "families": [
+                        "workfile"
+                    ],
+                    "hosts": [],
+                    "task_types": [],
+                    "tasks": [],
+                    "template": "{family}{Task}"
```

Comment:
shouldn't this rather be `task[name]`?

Reply:
I was thinking about it, but I found that we are still using only {Task} mostly. I think it would be good ideas to create issue to standardize.
(And figure out how to make it backward compatible, as {Task} will be prominent in customer's Settings, I am guessing.)